### PR TITLE
Add batch 1

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: meta/0436af4a.csv [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+71524,b15ae452,GitHub,0436af4a,data/0436af4a/test/b15ae452.cs,753,753,T,F,44,54,F,F,,,,,3.12,10,F,F,F,URL Credentials
+71525,c9fde51c,GitHub,0436af4a,data/0436af4a/test/c9fde51c.cs,97,97,T,F,16,23,F,F,,,,,2.81,7,F,F,F,Password
+71526,d197c0ec,GitHub,0436af4a,data/0436af4a/test/d197c0ec.cs,131,131,T,F,32,46,F,F,,,,,3.25,14,F,F,F,Password
+71527,d197c0ec,GitHub,0436af4a,data/0436af4a/test/d197c0ec.cs,132,132,T,F,32,46,F,F,,,,,3.05,14,F,F,F,Password
+71528,d2b2603a,GitHub,0436af4a,data/0436af4a/test/d2b2603a.cs,25,25,T,F,24,31,F,F,,,,,2.13,7,F,F,F,Key
+71529,d2b2603a,GitHub,0436af4a,data/0436af4a/test/d2b2603a.cs,150,150,T,F,29,37,F,F,,,,,2.75,8,F,F,F,Key
+71530,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,169,169,F,F,26,36,F,F,,,,,3.12,10,F,F,F,Password
+71531,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,170,170,F,F,26,37,F,F,,,,,3.28,11,F,F,F,Password
+109384,0f5653ad,GitHub,0436af4a,data/0436af4a/src/0f5653ad.cs,16,16,T,F,35,71,F,F,,,,,3.9329994,,F,F,F,Token
+109385,399829cf,GitHub,0436af4a,data/0436af4a/src/399829cf.cs,21,21,T,F,25,34,F,F,,,,,3.251629,,F,F,F,Password
```
Comment (babenek) [inline meta/0436af4a.csv:95]:
```suggestion
109385,399829cf,GitHub,0436af4a,data/0436af4a/src/399829cf.cs,21,21,T,F,25,35,F,F,,,,,3.251629,,F,F,F,Password
```

File: meta/0436af4a.csv [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+109390,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,212,212,F,F,20,37,F,F,,,,,3.8077636,,F,F,F,Token
```
Comment (babenek) [inline meta/0436af4a.csv:96]:
```suggestion
109390,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,212,212,F,F,,,F,F,,,,,3.8077636,,F,F,F,Token
```

File: meta/0436af4a.csv [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+71471,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,15,15,F,F,39,43,F,F,,,,,2.00,4,F,F,F,Auth
+71472,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,34,34,F,F,48,58,F,F,,,,,3.12,10,F,F,F,Credential
+71473,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,35,35,F,F,29,34,F,F,,,,,1.92,5,F,F,F,API
+71474,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,97,97,F,F,45,60,F,F,,,,,3.25,15,F,F,F,Auth
+71475,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,110,110,F,F,48,67,F,F,,,,,3.17,19,F,F,F,Auth
+71476,6d6c3e78,GitHub,0436af4a,data/0436af4a/src/6d6c3e78.cs,129,129,F,F,54,58,F,F,,,,,2.00,4,F,F,F,Auth
+71477,76af6e50,GitHub,0436af4a,data/0436af4a/src/76af6e50.cs,14,14,T,F,37,57,F,F,,,,,3.32,20,F,F,F,Auth
+71478,80495516,GitHub,0436af4a,data/0436af4a/src/80495516.cs,15,15,F,F,53,57,F,F,,,,,2.00,4,F,F,F,Auth
+71479,80495516,GitHub,0436af4a,data/0436af4a/src/80495516.cs,38,38,F,F,50,54,F,F,,,,,2.00,4,F,F,F,Auth
+71480,847e8b11,GitHub,0436af4a,data/0436af4a/src/847e8b11.cs,100,100,F,F,49,61,F,F,,,,,2.65,12,F,F,F,Credential
```
Comment (babenek) [inline meta/0436af4a.csv:42]:
```suggestion
71480,847e8b11,GitHub,0436af4a,data/0436af4a/src/847e8b11.cs,100,100,F,F,,,F,F,,,,,2.65,12,F,F,F,Credential
```

File: meta/0436af4a.csv [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+109395,e79ea97a,GitHub,0436af4a,data/0436af4a/src/e79ea97a.cs,162,162,F,F,37,73,F,F,,,,,3.7499413,,F,F,F,Token
+109396,114e3c56,GitHub,0436af4a,data/0436af4a/test/114e3c56.cs,19,19,F,F,42,55,F,F,,,,,3.7004397,,F,F,F,Token
+109397,114e3c56,GitHub,0436af4a,data/0436af4a/test/114e3c56.cs,22,22,F,F,43,56,F,F,,,,,3.7004397,,F,F,F,Token
+109399,61d643c5,GitHub,0436af4a,data/0436af4a/test/61d643c5.cs,16,16,T,F,33,69,F,F,,,,,3.7499413,,F,F,F,Token
+109400,697a0f01,GitHub,0436af4a,data/0436af4a/test/697a0f01.cs,16,16,T,F,25,61,F,F,,,,,3.8292398,,F,F,F,Token
+109401,4856a9ca,GitHub,0436af4a,data/0436af4a/test/4856a9ca.cs,610,610,T,F,25,61,F,F,,,,,3.5312107,,F,F,F,Token
+109402,92b8ee86,GitHub,0436af4a,data/0436af4a/test/92b8ee86.cs,208,208,T,F,19,55,F,F,,,,,3.7527153,,F,F,F,Token
+109403,92b8ee86,GitHub,0436af4a,data/0436af4a/test/92b8ee86.cs,80,80,F,F,29,38,F,F,,,,,2.6635327,,F,F,F,Password
+109406,d5419004,GitHub,0436af4a,data/0436af4a/test/d5419004.cs,71,71,F,F,25,33,F,F,,,,,2.9219282,,F,F,F,Password
+109407,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,119,119,F,F,26,37,F,F,,,,,3.3927474,,F,F,F,Password
```
Comment (babenek) [inline meta/0436af4a.csv:106]:
```suggestion
109407,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,119,119,T,F,26,37,F,F,,,,,3.3927474,,F,F,F,Password
```

File: meta/0436af4a.csv [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+71521,8a1f6cd5,GitHub,0436af4a,data/0436af4a/test/8a1f6cd5.cs,226,226,T,F,28,35,F,F,,,,,2.52,7,F,F,F,Password
+71522,92b8ee86,GitHub,0436af4a,data/0436af4a/test/92b8ee86.cs,209,209,T,F,23,42,F,F,,,,,3.22,19,F,F,F,Secret
+71523,b15ae452,GitHub,0436af4a,data/0436af4a/test/b15ae452.cs,751,751,T,F,33,43,F,F,,,,,3.12,10,F,F,F,Password
+71524,b15ae452,GitHub,0436af4a,data/0436af4a/test/b15ae452.cs,753,753,T,F,44,54,F,F,,,,,3.12,10,F,F,F,URL Credentials
+71525,c9fde51c,GitHub,0436af4a,data/0436af4a/test/c9fde51c.cs,97,97,T,F,16,23,F,F,,,,,2.81,7,F,F,F,Password
+71526,d197c0ec,GitHub,0436af4a,data/0436af4a/test/d197c0ec.cs,131,131,T,F,32,46,F,F,,,,,3.25,14,F,F,F,Password
+71527,d197c0ec,GitHub,0436af4a,data/0436af4a/test/d197c0ec.cs,132,132,T,F,32,46,F,F,,,,,3.05,14,F,F,F,Password
+71528,d2b2603a,GitHub,0436af4a,data/0436af4a/test/d2b2603a.cs,25,25,T,F,24,31,F,F,,,,,2.13,7,F,F,F,Key
+71529,d2b2603a,GitHub,0436af4a,data/0436af4a/test/d2b2603a.cs,150,150,T,F,29,37,F,F,,,,,2.75,8,F,F,F,Key
+71530,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,169,169,F,F,26,36,F,F,,,,,3.12,10,F,F,F,Password
```
Comment (babenek) [inline meta/0436af4a.csv:92]:
```suggestion
71530,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,169,169,T,F,26,36,F,F,,,,,3.12,10,F,F,F,Password
```

File: meta/0436af4a.csv [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+71531,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,170,170,F,F,26,37,F,F,,,,,3.28,11,F,F,F,Password
```
Comment (babenek) [inline meta/0436af4a.csv:93]:
```suggestion
71531,df627f1b,GitHub,0436af4a,data/0436af4a/test/df627f1b.cs,170,170,T,F,26,37,F,F,,,,,3.28,11,F,F,F,Password
```

File: meta/0436af4a.csv [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+71460,c7cee009,GitHub,0436af4a,data/0436af4a/other/c7cee009.md,52,52,T,F,17,27,F,F,,,,,3.12,10,F,F,F,Password
+71461,031868ed,GitHub,0436af4a,data/0436af4a/src/031868ed.axaml,5,5,F,F,75,82,F,F,,,,,2.52,7,F,F,F,Credential
+71462,0a5f84be,GitHub,0436af4a,data/0436af4a/src/0a5f84be.axaml,5,5,F,F,68,75,F,F,,,,,2.52,7,F,F,F,Credential
+71463,0f5653ad,GitHub,0436af4a,data/0436af4a/src/0f5653ad.cs,27,27,F,F,43,48,F,F,,,,,2.32,5,F,F,F,Credential
+71464,b8e838d1,GitHub,0436af4a,data/0436af4a/src/b8e838d1.cs,153,153,F,F,18,48,F,F,,,,,3.52,30,F,F,F,Password
+71465,2570e05e,GitHub,0436af4a,data/0436af4a/src/2570e05e.cs,189,189,F,F,58,67,F,F,,,,,2.24,9,F,F,F,Auth
+71466,b470d7e1,GitHub,0436af4a,data/0436af4a/src/b470d7e1.axaml,5,5,F,F,68,75,F,F,,,,,2.52,7,F,F,F,Credential
+71467,ec08ec22,GitHub,0436af4a,data/0436af4a/src/ec08ec22.axaml,6,6,F,F,76,83,F,F,,,,,2.52,7,F,F,F,Credential
+71468,aac28416,GitHub,0436af4a,data/0436af4a/src/aac28416.cs,13,13,F,F,32,49,F,F,,,,,3.57,17,F,F,F,API
+71469,ef52c2b8,GitHub,0436af4a,data/0436af4a/test/ef52c2b8.cs,73,73,F,F,25,35,F,F,,,,,3.12,10,F,F,F,Password
```
Comment (babenek) [inline meta/0436af4a.csv:31]:
```suggestion
71469,ef52c2b8,GitHub,0436af4a,data/0436af4a/test/ef52c2b8.cs,73,73,T,F,25,35,F,F,,,,,3.12,10,F,F,F,Password
```

**Review by babenek — COMMENTED** [review]
Suggested changes

**Review by babenek — APPROVED** [review]
Good. MIT license

**Review by babenek — APPROVED** [review]
Approved

**Review by iuriimet — APPROVED** [review]

**Pull request merged**